p02.lisp

```diff
@@ -6,7 +6,7 @@
 
 (defun my-but-last (lst)
   (if (null (cddr lst))
-      lst
```

Comment:
Hi @vinodkumar2704, I think the syntax you want is `(car lst)`, but then the function won't pass the test. If you want to run this code, I suggest installing a common lisp implementation (sbcl is a good choice, but any common lisp should be ok) and [quicklisp](https://www.quicklisp.org/beta/).

After installing quicklisp, clone this repo into ~/quicklisp/local-projects (or wherever you installed quicklisp), then start up your lisp and run `(ql:quickload :99-problems)`. That should load this package and its dependencies. You can then run the test defined in this file like so:

```
CL-USER> (in-package :99-problems)
#<PACKAGE "99-PROBLEMS">
99-PROBLEMS> (run-tests '(my-but-last-test))
Unit Test Summary
 | 1 assertions total
 | 1 passed
 | 0 failed
 | 0 execution errors
 | 0 missing tests

#<TEST-RESULTS-DB Total(1) Passed(1) Failed(0) Errors(0)>
```

You can instead run all the tests with `(run-tests)`.

Good luck & happy hacking!